app/src/main/java/org/wikipedia/edits/EditsTypeItem.kt

```diff
@@ -10,10 +10,10 @@ import androidx.core.content.ContextCompat
 import androidx.core.widget.ImageViewCompat
 import kotlinx.android.synthetic.main.item_suggested_edits_type.view.*
 import org.wikipedia.R
-import org.wikipedia.suggestededits.Contribution.Companion.EDIT_TYPE_GENERIC
+import org.wikipedia.edits.Contribution.Companion.EDIT_TYPE_GENERIC
 import org.wikipedia.util.ResourceUtil
 
-class SuggestedEditsTypeItem constructor(context: Context, attrs: AttributeSet? = null) : LinearLayout(context, attrs) {
+class EditsTypeItem constructor(context: Context, attrs: AttributeSet? = null) : LinearLayout(context, attrs) {
```

Comment:
Since this class is a View, the name of the class should probably end in `...View`, to keep consistent with our other Views.  e.g. `EditTypeItemView`.